src/metadata/metadata.proto

```diff
@@ -7,6 +7,10 @@ import "pfs/pfs.proto";
 
 import "protoextensions/validate.proto";
 
+// ClusterPicker selects a cluster.  Since clusters will never "cascade", there is only one cluster
+// that can be selected, the one running this API server.
+message ClusterPicker {}
```

Comment:
I think we can freely add to this message if the need arrives.  Right now, we just need a sentinel to pick a cluster instead of a commit or project.  I choose a message for future extensibility, so it's `"cluster":{}` instead of `"cluster": true` (which doesn't have the extensibility).  See the reply to a comment above about the binary representation; a oneof being set to an empty message is different than omitting the oneof, so this is an unambiguous request that you do want to target the cluster.